src/rabbit_auth_backend_ldap.erl

```diff
@@ -314,14 +323,16 @@ with_login(Creds, Servers, Opts, Fun) ->
                 {UserDN, Password} ->
                     case eldap:simple_bind(LDAP, UserDN, Password) of
                         ok ->
-                            ?L1("bind succeeded: ~s", [UserDN]),
+                            ?L1("bind succeeded: ~s",
```

Comment:
@hairyhum Okay cool. I'm extending this to also be scrubbed under `true` log verbosity. So ONLY under `network_unsafe` would credentials be exposed, and in the minimal LDAP decision logs, we'd still have credentials/DN in that bind outcome log entry, scrubbed by default. Makes sense. Thanks! 👍